app/models/editionable_worldwide_organisation.rb

```diff
@@ -1,4 +1,5 @@
 class EditionableWorldwideOrganisation < Edition
```

Comment:
tiny comment on the commit message for this overflowing

Reply:
Have fixed that